includes/discovery/os/nios.inc.php

```diff
@@ -0,0 +1,10 @@
+<?php
+if (!$os || $os === 'linux') {
+    if (stristr($sysObjectId, ".1.3.6.1.4.1.7779.1")) {
+        $os = 'nios';
+    }
+}
+
```

Comment:
I'll be around his work soon and try out the PR on his gear. Will do some fixes from there if needed.
Looking forward to another device being added :+1:

On 19 January 2016 11:31:51 CET, Neil Lathwood <notifications@github.com> wrote:
>> @@ -0,0 +1,10 @@
>> +<?php
>> +if (!$os || $os === 'linux') {
>> +    if (stristr($sysObjectId, ".1.3.6.1.4.1.7779.1")) {
>> +        $os = 'nios';
>> +    }
>> +}
>> +
>
>Lots of blank lines here, can you remove those please.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/librenms/librenms/pull/2801/files#r50096133